canonical-websites/www.ubuntu.com
# Add k8s docs pages

## Done
Use pandoc to convert the [current Kubernetes docs](https://blog.juju.solutions/kubernetes-docs/) into www.ubuntu.com template pages. 

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /kubernetes/docs
- Click through the navigation and see the pages load and look good

## Issue / Card
Fixes #4344
